CoreMentoringApp.ConsoleRestClient/Program.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using CoreMentoringApp.ConsoleRestClient.Models;
+using Newtonsoft.Json;
+
+namespace CoreMentoringApp.ConsoleRestClient
+{
+    class Program
+    {
+        static HttpClient client = new HttpClient();
+
+        static void Main(string[] args)
+        {
+            RunAsync().GetAwaiter().GetResult();
```

Comment:
Starting with C# 7.1 your Main() can be marked as async: `static async Task Main()` and you will be able to use `await` directly inside Main() https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-7#async-main

Reply:
Good point, thanks